src/pages/add-lock-exception/AddLockExceptionForm/OrganisationUnitSelection/SelectByGroup.js

```diff
@@ -0,0 +1,111 @@
+import { useDataEngine, useAlert } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
+import PropTypes from 'prop-types'
+import React, { useState } from 'react'
+import Controls from './Controls'
+import useOrgUnitCache from './use-org-unit-cache'
+
+const currentRootOrgUnitsQuery = {
+    orgUnits: {
+        resource: 'organisationUnits',
+        id: ({ currentRootId }) => currentRootId,
+        params: ({ groupId }) => ({
+            fields: 'id,path',
+            filter: `organisationUnitGroups.id:eq:${groupId}`,
+            includeDescendants: true,
+            paging: false,
+        }),
+    },
+}
+const orgUnitGroupsQuery = {
+    orgUnitGroups: {
+        resource: 'organisationUnitGroups',
+        id: ({ groupId }) => groupId,
+        params: {
+            fields: `organisationUnits[id,path]`,
+        },
+    },
+}
+
+const SelectByGroup = ({ groups, currentRootId, onSelect, onDeselect }) => {
```

Comment:
This component is a rewrite of https://github.com/dhis2/d2-ui/blob/master/packages/org-unit-select/src/OrgUnitSelectByGroup.component.js